src/frontend/src/flows/register/passkey.ts

```diff
@@ -10,26 +10,39 @@ import {
   isWebAuthnCancel,
   webAuthnErrorCopy,
 } from "$src/utils/webAuthnErrorUtils";
-import { html, TemplateResult } from "lit-html";
+import { html, nothing, TemplateResult } from "lit-html";
```

Comment:
this is new / not used anywhere else in the code base, for consistency reason is there a way to solve the requirement in another way?

Reply:
The canonical way seems to be `${condition ? element : ""}` so I have changed it to that.